Source/WebCore/html/HTMLFormControlElement.h

```diff
@@ -95,6 +95,11 @@ class HTMLFormControlElement : public HTMLElement, public ValidatedFormListedEle
 
     virtual String resultForDialogSubmit() const;
 
+    HTMLElement* popoverTargetElement() const;
```

Comment:
Do these really all need to be public?

Reply:
Yes, for the IDL generated code.

Reply:
Really, how about handlePopoverTargetAction?

Reply:
Fixed